llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -5930,6 +5977,10 @@ SDValue NVPTXTargetLowering::PerformDAGCombine(SDNode *N,
     return combineADDRSPACECAST(N, DCI);
   case ISD::AND:
     return PerformANDCombine(N, DCI);
+  case ISD::ANY_EXTEND:
```

Comment:
Is there any test cases which cover the ANY_EXTEND case? If we can't construct IR to hit this case, I'd just go ahead and remove it.